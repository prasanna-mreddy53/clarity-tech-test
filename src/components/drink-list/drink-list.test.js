import React from "react";
import { shallow } from "enzyme";
import DrinkList from "./drink-list.component";

const setup = (props) => {
  const component = shallow(<DrinkList {...props} />);
  return component;
};

describe("Drinks Component Tests", () => {
  const props = {
    drink: {
      strDrink: "410 Gone",
      idDrink: "13581",
      strDrinkThumb:
        "https://www.thecocktaildb.com/images/media/drink/xtuyqv1472669026.jpg",
    },
  };

  let drinks = setup(props);

  it("renders correctly", () => {
    expect(drinks).toMatchSnapshot();
  });
});
