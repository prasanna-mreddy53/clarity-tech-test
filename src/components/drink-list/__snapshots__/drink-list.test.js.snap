// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Drinks Component Tests renders correctly 1`] = `ShallowWrapper {}`;
